### PR TITLE
Rails 5.0 support

### PR DESCRIPTION
**Pull request by holidayworking**
#295 is fixed.

### CODE REVIEW

File: lib/apartment/adapters/mysql2_adapter.rb [hunk 1]
```diff
@@ -34,6 +34,27 @@ def initialize(config)
...
+
+          switch(tenant) do
+            # Seed data if appropriate
+            seed_data if Apartment.seed_after_create
+
+            yield if block_given?
+          end
+        end
+      end
+
```
Comment (mikecmpbll) [inline lib/apartment/adapters/mysql2_adapter.rb]: what's the reason for this?

**Comment by mikecmpbll** [comment]
looking at the test fails, it seems to be related to the new `ar_internal_metadata` system table in Rails 5. maybe we need to include the create syntax for that in the schema? *shrugs

**Comment by milep** [comment]
Found explanation for the `ar_internal_metadata` table: http://blog.bigbinary.com/2016/06/07/rails-5-prevents-destructive-action-on-production-db.html.
Maybe set DISABLE_DATABASE_ENVIRONMENT_CHECK=1 environment variable as temporary hack?

**Comment by bradrobertson** [comment]
I've just merged in [this](https://github.com/influitive/apartment/commit/2b882d5910b25157d7ecff57bd91509bb2edae66) and [this](https://github.com/influitive/apartment/commit/83770b1452b1a6d0ecfb044d3807e897b66c8eb1) which (AFAIK) should be good for rails 5 support. Not sure why your tests were failing, but it worked fine for me. 
Is there any reason I shouldn't release this? We don't use mysql so not sure what that change above might be for.

**Comment by mikecmpbll** [comment]
i don't think this PR is required now.

p.s. i think the gemfiles for travis are broken, but i think they always have been. rails 4.0 one has this for instance:

`gem "rails", "~> 4.0"`

the squiqqly operator will evaluate to the highest version after the last dot? so this will actually use 4.2 in travis i believe, same with ther 4.1 one. should be ~> 4.0.0 and ~> 4.1.0, etc.

> ~> 2.1 is identical to >= 2.1 and < 3.0
http://bundler.io/gemfile.html

i'll do a PR now tho and hopefully it doesn't reveal any test fails 😁😓

**Comment by bradrobertson** [comment]
Ok I'm going to close this then and release 1.2 for rails 5.

**Pull request closed without merging**